docs/quality-standards/scorecards.yaml

```diff
@@ -1084,10 +1083,8 @@ Tecken:
           LOGGING_DEFAULT_LEVEL env var
           https://github.com/mozilla-services/tecken/blob/3543f9d39f12cd65da1539341692663fd4621778/tecken/settings.py#L87-L91
       monitoring-error-reporting:
-        compliant: No
-        notes:
-          https://github.com/mozilla-services/tecken/blob/3543f9d39f12cd65da1539341692663fd4621778/bin/sentry-wrap.py#L43C21-L43C31  Not
-          environment / version passed to init
+        compliant: Yes
+        notes: Uses fillmore
```

Comment:
sentry-wrap is a script for wrapping migrations during deploy and things like that. The Tecken webapp uses fillmore to set up Sentry:

https://github.com/mozilla-services/tecken/blob/ddde451cff69026e8f6c479e6242dc051c6578d8/tecken/apps.py#L66-L93